actions/register.php

```diff
@@ -3,20 +3,20 @@
  * Elgg registration action
  */
 
+/* @var $request \Elgg\Request */
```

Comment:
Where did this come from? Is `$request` always available in actions, like `$vars` in views?

Reply:
Yes, it's available in scope when action file is included.
In resource views you can access request through `$vars['request']`

Reply:
Switching to using request now, will make it easier to transition to action classes later